todo_app/app.py

```diff
@@ -0,0 +1,78 @@
+from flask import Flask, render_template, request, redirect, url_for
+import todo_app.data.session_items as session
+from todo_app.flask_config import Config
+import requests
+def create_app():
+    app = Flask(__name__)
+    app.config.from_object(Config)
+    # All the routes and setup code etc
+
+    @app.route('/')
+    def index():  
+        try:        
+            #return "hello"              
+            items = session.get_items()
+            item_view_model = session.ViewModel(items)
+            return render_template('index.html',view_model=item_view_model)
+            
+        except Exception as e:
+            return str(e)
```

Comment:
This removes the stack trace making this harder to debug

Reply:
Could you point mean the right direction as I don't understand what you mean here

Reply:
please I need to understand how this removes stack trace